src/Behat/Behat/HelperContainer/Exception/NotFoundException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\HelperContainer\Exception;
+
+class_alias(
+    interface_exists('Interop\\Container\\Exception\\NotFoundException')
+        ? 'Interop\\Container\\Exception\\NotFoundException'
+        : 'Psr\\Container\\NotFoundExceptionInterface',
+    'Behat\\Behat\\HelperContainer\\Exception\\NotFoundException'
+);
+
+if (false) {
+    interface NotFoundException
```

Comment:
I suggest adding a phpdoc on these fake interface to tag them as `@internal`. We don't want external packages to use them.